AnagramTS_/src/anagram.ts

```diff
@@ -11,11 +11,11 @@ export function fillMatchedWords() {
   const inputString: string = sorting("melon");
   const inputStringLength: number = inputString.length;
 
-  const wordListLength:number = words.length;
+  const wordListLength: number = words.length;
   const matchedWordsList: string[] = [];
 
   for (let i = 0; i < wordListLength; i++) {
```

Comment:
you don't actually use the index for anything so please use the more modern approach to run through the words elements